### ISSUE TITLE
how to implement adaptive-softmax using tensorflow?

### ISSUE THREAD

**Issue by OswaldoBornemann** [facebookresearch/adaptive-softmax#6]
how to implement adaptive-softmax while using tensorflow? It seems that tensorflow does not have the api.

**Comment by ricardobnjunior** [comment]
Same problem.

**Comment by bywords** [comment]
It seems the issue has been opened for a while, but this pointer below might be of help to those who are looking for the same:
https://github.com/yangsaiyong/tf-adaptive-softmax-lstm-lm
